cmd/updater/main.go

```diff
@@ -160,7 +163,7 @@ func main() {
 	})
 	log.Info("Configured concurrency")
 
-	groupUpdater := updater.GCS(ctx, client, opt.groupTimeout, opt.buildTimeout, opt.buildConcurrency, opt.confirm)
+	groupUpdater := updater.GCS(ctx, client, opt.groupTimeout, opt.buildTimeout, opt.buildConcurrency, opt.confirm, opt.enableIgnoreSkip)
```

Comment:
not required for this PR. I see a lot of crucial methods are being called with increasing number of parameters, maybe we should aggregating those and pass the aggregated structs of params

Reply:
Agreed! Filed https://github.com/GoogleCloudPlatform/testgrid/issues/1121 to track ^^